src/css/toc.css

```diff
@@ -22,9 +23,9 @@
 }
 
 .toc .toc-menu h3 {
-  font-size: 0.9375rem;
-  font-weight: var(--body-font-weight-bold);
+  font-size: 0.777rem;
   margin: 0;
+  text-transform: uppercase;
```

Comment:
Why are we using uppercase? Uppercase might "look cool" at times but reduces readability significantly.

Reply:
We talked about this in slack, but if you have more questions about the design motivation, they may get more answers in the [figma](https://www.figma.com/file/768l7kxM3F3YLvHoTBnAhz/SDK-2021?node-id=419%3A20).